### PR TITLE
feature/service card

### PR DESCRIPTION
**Pull request by phongduong-dp**
# Description & Technical Solution

ServiceCard component used for FeatureList later

# Checklist

- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] Already rebased against main branch.

### CODE REVIEW

File: playground/src/ServiceCard.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  >
+    <ServiceCard content={content} />{" "}
+  </div>
+);
+
+export const NoContent: Story = () => (
+  <div
+    style={{
+      width: "100%",
+      padding: "20px",
```
Comment (its-rav) [inline playground/src/ServiceCard.stories.tsx:42]: please use theme

File: playground/src/ServiceCard.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      backgroundColor: "black",
```
Comment (its-rav) [inline playground/src/ServiceCard.stories.tsx:43]: please use theme

File: packages/core-components/src/components/buttons/Button.tsx [hunk 3]
```diff
@@ -47,9 +47,13 @@ const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
             <ImSpinner2 className='animate-spin' />
           </div>
         )}
-        {leftIcon && <div>{leftIcon}</div>}
+        {leftIcon && (
+          <div className='md:w-3.5 md:h-3.5 sm:w-2.5 sm:h-2.5'>{leftIcon}</div>
```
Comment (its-rav) [inline packages/core-components/src/components/buttons/Button.tsx:51]: sao số lẻ z ta

File: packages/core-components/src/components/ServiceCard.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+          <div className='md:w-14 md:h-14 sm:w-10 sm:h-10'>{content.icon}</div>
+          <p
+            className={`mt-3 md:text-xl font-bold md:leading-6 sm:text-sm text-primary ${titleClassName}`}
+          >
+            {content.title}
+          </p>
+          <p className={`${descriptionClassName}`}>{content.description}</p>
+        </>
+      )}
+      {children}
```
Comment (its-rav) [inline packages/core-components/src/components/ServiceCard.tsx:39]: is this `children` prop necessary?

File: packages/core-components/src/components/ServiceCard.tsx [hunk 5]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+export type ServiceCardContentType = {
+  icon: React.ReactNode;
+  title: string;
+  description: string;
+};
+
+type ServiceCardType = {
+  content?: ServiceCardContentType;
```
Comment (its-rav) [inline packages/core-components/src/components/ServiceCard.tsx:10]: why is this nullable

File: packages/core-components/src/components/sections/StatisticSection.tsx [hunk 6]
```diff
@@ -1,13 +1,13 @@
...
 
-export type ContentType = {
+export type StatisticContentType = {
   title: string;
   statistic: string;
 };
 
 export type StatisticSectionProps = {
-  content: ContentType[];
+  content: StatisticContentType[];
```
Comment (its-rav) [inline packages/core-components/src/components/sections/StatisticSection.tsx:10]: những cái này nên naming theo kiểu list cho nó tường minh nha fren, ...Items hay ...Types gì đó

**Review by its-rav — CHANGES_REQUESTED** [review]
pls check these comment and resolve the conflict

**Comment by phongduong-dp** [comment]
@its-rav please review, tailwind config has been modified, children prop is deleted, "content" has been changed to "serviceCardItem" for better understanding.

File: packages/core-components/src/components/ServiceCard.tsx [hunk 7]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+export type ServiceCardItemType = {
+  icon: React.ReactNode;
+  title: string;
+  description: string;
+};
+
+type ServiceCardType = {
```
Comment (its-rav) [inline packages/core-components/src/components/ServiceCard.tsx:9]: should use the same convention {component}Props, or {component}Type if you want. But follow one only

**Review by its-rav — APPROVED** [review]
Please update the component according to the comment
Then it's good to go

**Pull request merged**